tests/modules/programs/bemenu/basic-configuration.nix

```diff
@@ -0,0 +1,25 @@
+{
+  programs.bemenu = {
+    enable = true;
+    settings = {
+      line-height = 28;
+      prompt = "open";
+      fb = "#1e1e2e";
+      ff = "#cdd6f4";
+      nb = "#1e1e2e";
+      nf = "#cdd6f4";
+      tb = "#1e1e2e";
+      hb = "#1e1e2e";
+      tf = "#f38ba8";
+      hf = "#f9e2af";
+      af = "#cdd6f4";
+      ab = "#1e1e2e";
+    };
+  };
+
+  nmt.script = ''
+    assertFileExists home-path/etc/profile.d/hm-session-vars.sh
+    assertFileContent home-path/etc/profile.d/hm-session-vars.sh \
+      ${./basic-configuration.sh}
```

Comment:
This will lead to needless churn if the surrounding content of `hm-session-vars.sh` changes. You should probably use `assertFileContains` instead.

Reply:
I've updated the tests to use `assertFileContains` and `assertFileNotRegex` instead. Let me know if there is a better way to handle this.